app/templates/notify_template.html

```diff
@@ -22,5 +22,8 @@
   </head>
   <body>
     {% block main_content %}{% endblock %}
+    <script src="{{ asset_path }}javascripts/main.min.js"></script>
```

Comment:
```suggestion
    <script src="{{ asset_path }}javascripts/main.min.js"></script>
```
You already include this file afterwards, I think you can delete that

Reply:
My mistake, I'll remove.